### DISCUSSION TITLE
Does kafkahq support authorization code flow with proof key for code Exchange (PKCE)?

### DISCUSSION THREAD

**Discussion by swapnilborade** [tchiotludo/akhq#734] [Q&A]
Does kafkahq support authorization code flow with proof key for code Exchange (PKCE)?

**Comment by tchiotludo** [comment]
Really don't know. 
Never heard about that but we really on micronaut for this, take a look [at this](https://micronaut-projects.github.io/micronaut-security/latest/guide/#oauth)
